app/models/device_problem.rb

```diff
@@ -4,6 +4,14 @@ class DeviceProblem < ApplicationRecord
 
   validates :statement, :content, :device_id, presence: true
 
+  validate :statement_within_allowed_length, if: :statement
```

Comment:
There's a build in length validator:
```ruby
validates :statement, length: { less_than_or_equal_to: 35, message: 'must contain less than 35 characters' }
```
Double check syntax.. and maybe what the default error message it produces as you might not need to set a custom message either.